### DISCUSSION TITLE
才发现家里猪咪这么肥了

### DISCUSSION THREAD

**Discussion by LeoneMafia** [share121/inter-knot#5403] [灌水]
![mmexport1726319951409](https://github.com/user-attachments/assets/672843e6-e02d-44c0-ab4e-4eaed45af50a)
![IMG20240819151059](https://github.com/user-attachments/assets/b581dbfe-ded6-44e0-adb9-304e2aa39d3d)
咱就是说，橘猫都这么容易发福的吗？

**Comment by Robin-txdy** [comment]
好可爱

**Reply by LeoneMafia** [reply]
是可爱，就是天天咬人

**Comment by XinRenShenJiangZiShu** [comment]
十分的圆润

**Reply by LeoneMafia** [reply]
这下真成猪咪了

**Comment by aomengxin** [comment]
加菲猫罢

**Reply by LeoneMafia** [reply]
这是纯正的大橘，而且加菲猫可不会天天咬人
